moveit_ros/moveit_servo/include/moveit_servo/utils/command.hpp

```diff
@@ -47,6 +47,10 @@
 #include <rclcpp/rclcpp.hpp>
 #include <tf2_eigen/tf2_eigen.hpp>
 
+#define COMMAND_ERROR(msg_stream)                                                                                      \
```

Comment:
Would strongly advise against macros since it makes things hard to debug.

But just put the error + return lines in the code itself and it looks good!